src/scoring/processor/expressions/operators/max.js

```diff
@@ -62,7 +62,7 @@ var maxProcessor = {
             return null;
         }
 
-        result.value = castedOperands.max().value();
+        result.value = castedOperands.max();
```

Comment:
question: why remove `.value()` call?

Reply:
Because `value` is a mathod of extracting value from lodash-wrapped object. `max` returns value, not the wrapper objects [max ](https://lodash.com/docs/4.17.15#max)